pkg/ebpf/processor.go

```diff
@@ -130,6 +121,16 @@ func (t *Tracee) registerEventProcessors() {
 	t.RegisterEventProcessor(events.PrintNetSeqOps, t.processTriggeredEvent)
 	t.RegisterEventProcessor(events.PrintMemDump, t.processTriggeredEvent)
 	t.RegisterEventProcessor(events.PrintMemDump, t.processPrintMemDump)
+
+	//
+	// Event Timestamps Normalization Processors
+	//
+
+	// NOTE: Make sure to convert time related args (of your event) BELOW HERE:
+	t.RegisterEventProcessor(events.SchedProcessFork, t.processSchedProcessFork)
+
+	// Keep this last, orelse it will override event context times before other "All" processors.
+	t.RegisterEventProcessor(events.All, t.normalizeEventCtxTimes)
```

Comment:
Please move the DNS Cache processor above this.

Reply:
Check my comment below.

Reply:
So, DNS cache IS above it. Assuming you meant "below" instead (because you need the non normalized time in the DNS cache), then I you can use `Tracee.getOrigEvtTimestamp()` and revert the timestamp for your specific needs (some other stages later in the pipeline do that).